flow/serialize.h

```diff
@@ -634,14 +634,19 @@ class _Reader {
 		check = nullptr;
 	}
 
+	size_t size() const { return len; }
```

Comment:
Should this be changed to `originalSize`? It looks like only the `remainingBytes` function may be necessary.

Reply:
Maybe deleting the method is acceptable due to is not used. I added it because I predict we'll need it in the future (not sure though